src/docfx/build/xref/XrefMapBuilder.cs

```diff
@@ -166,7 +166,7 @@ private static (List<Error> errors, IReadOnlyList<InternalXrefSpec> specs) LoadS
                 var xref = new InternalXrefSpec
                 {
                     Uid = item.Key,
-                    Href = isRoot ? file.CanonicalUrlWithoutLocale : $"{file.CanonicalUrlWithoutLocale}#{GetBookmarkFromUid(item.Key)}",
+                    Href = isRoot ? file.SiteUrl : $"{file.SiteUrl}#{GetBookmarkFromUid(item.Key)}",
```

Comment:
remove `CanonicalUrlWithoutLocale` from document

Reply:
it is still in use when outputting xref map